### ISSUE TITLE
Journey Map - Benefit Analyst

### ISSUE THREAD

**Issue by harrydchand** [CodeForPhilly/benefit-decision-toolkit#115]
Create a Journey Map for the Benefits Analyst Experience
UX Research

**Comment by harrydchand** [comment]
@Zionlove-a11y  I am unable to make this appear on the KanBan boards. I the problem is that there is no "Project" set.
However when I try to set a Project on the issue right side all the projects are greyed out.

<img width="473" height="520" alt="Image" src="https://github.com/user-attachments/assets/95ae51dd-76ee-4573-a130-57f82145213c" />
